lib/measured/units/weight.rb

```diff
@@ -1,6 +1,10 @@
 Measured::Weight = Measured.build do
   unit :g, aliases: [:gram, :grams]
   unit :kg, value: "1000 g", aliases: [:kilogram, :kilograms]
+  unit :T, value: "1000 kg", aliases: [:tonne, :tonnes]
+  unit :slug, value: "14.593903 kg", aliases: [:slugs]
+  unit :N, value: "9.80665 kg", aliases: [:newtons, :newton]
```

Comment:
I think this is reversed from what I can tell searching the internet. 1 kilogram is equal to 9.80665 newtons. This may be why you're seeing a discrepancy below (which you've commented on).